manage_breast_screening/notifications/management/commands/send_message_batch.py

```diff
@@ -87,6 +89,14 @@ def mark_batch_as_failed(self, message_batch: MessageBatch, response: Response):
         message_batch.notify_errors = response.json()["errors"]
         if response.status_code in RECOVERABLE_STATUS_CODES:
             message_batch.status = "failed_recoverable"
```

Comment:
We define a bunch of MessageBatch statuses in models: https://github.com/NHSDigital/dtos-manage-breast-screening/blob/main/manage_breast_screening/notifications/models.py#L7-L12 perhaps we can add these new failure statuses too.